### ISSUE TITLE
Add description on how translation works and how they could be checked automatically to stringtables folder

### ISSUE THREAD

**Issue by Cjkjvfnby** [freeorion/freeorion#3524]
Adding a bit of documentation will be a bit useful. 
It is really hard to find information about st-tool and its usage.  

This issue was raised by @Grummel7 . I agree with his suggestion to add the README.md file to the translation folder.
https://github.com/freeorion/freeorion/pull/3513#issuecomment-877451466

**Comment by Vezzra** [comment]
@Cjkjvfnby, why did you assign this to the "Gateway to the Void" milestone? That milestone is only intended for closed rejected issues/PRs, not for valid open ones...?

**Comment by st-pa-fo** [comment]
this is outdated / obsolete due to #3652

**Comment by Cjkjvfnby** [comment]
> why did you assign this to the "Gateway to the Void" milestone? 

Probably because I don't remember the meaning of the releases, and try to guess by names.  I think I was looking into something like: "any future release"

**Comment by Vezzra** [comment]
@Cjkjvfnby, the rules for milestones are as follows:

- If the PR should be part of the next/upcoming release, assign it to the respecitve milestone (currently "v0.5")
- If it's not yet decided if the PR should be part of the next/upcoming release, or if the PR should *not* be part of the next/upcoming release, then no milestone must be set
- If the PR has been closed without merging (because it has been rejected, superseded, abandoned, etc.), then the "Gateway to the Void" milestone must be assigned (that milestone marks PRs that were not/must not be merged)

HTH 😃

**Comment by Vezzra** [comment]
@Cjkjvfnby, @st-pa-fo,

> this is outdated / obsolete due to #3652

Ok, @Cjkjvfnby set this to the 0.5 milestone, but @st-pa-fo says this PR has been superseded by #3652 - what is it? Still relevant or obsolete/superseded?

**Comment by Grummel7** [comment]
Since I raised the topic originally, I must say #3652 partially missed my point.
There is a lot of useful information in the readme, but it is in the folder of the st-tool.
When I started editing en.txt, I found the file easily, but I did not find the st-tool, nor any hint that there even was such a tool. That's why I suggested adding a readme to default/stringtables.

But that can be solved quickly, by adding a mini-readme that just tells where to find the information.
